runtime/interpreter/value.go

```diff
@@ -56,11 +56,9 @@ type typeConformanceResultEntry struct {
 //
 // NOTE: Do not generalize to map[interpreter.Value],
 // as not all values are Go hashable, i.e. this might lead to run-time panics
-//
```

Comment:
This is unfortunately a behaviour of Go 1.19's gofmt. We should maybe clean this up in a separate PR for the whole codebase

Reply:
https://github.com/golang/go/issues/40028

W O W you can't even change this behaviour